test/unit/mock/cmd/docker-ps-a

```diff
@@ -1,4 +1,4 @@
-{"Command":"\"/bin/bash\"","CreatedAt":"2017-04-24 10:29:12 +0200 CEST","ID":"3def9aa450f8bd772c3d5b07e27ec934e5f58575e955367a0aca2d93e0687536","Image":"ubuntu:12.04","Labels":"","LocalVolumes":"0","Mounts":"","Names":"sleepy_khorana","Networks":"bridge","Ports":"","RunningFor":"29 minutes","Size":"0 B","Status":"Exited (127) 2 seconds ago"}
```

Comment:
Seems like it might be a good idea to *keep* a test case in which there are no labels.

Reply:
All the other containers do not have labels and there are tests for them.

> NOTE: Docker does not return valid JSON but instead a JSON for each result so this may explain things, see the file for more context.